sentry/resource_sentry_issue_alert.go

```diff
@@ -54,9 +54,10 @@ func resourceSentryIssueAlertSchema() map[string]*schema.Schema {
 			Required:    true,
 		},
 		"name": {
-			Description: "The issue alert name.",
-			Type:        schema.TypeString,
-			Required:    true,
+			Description:  "The issue alert name.",
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringLenBetween(1, 63),
```

Comment:
The max should be 64 inclusive.

```suggestion
			ValidateFunc: validation.StringLenBetween(1, 64),
```

Reply:
Done :+1: